spec/features/merchants/statistics_spec.rb

```diff
@@ -34,14 +53,13 @@
       within ".merchant-stats" do
         expect(page).to have_content("Number of Items: 2")
         expect(page).to have_content("Average Price of Items: $15")
-        within ".distinct-cities" do
-          expect(page).to have_content("Cities that order these items:")
-          expect(page).to have_content("Hershey")
-          expect(page).to have_content("Denver")
-        end
       end
 
-
+      within ".distinct-cities" do
```

Comment:
Did you have a failing test due to the way this part was positioned in the test? It seemed that the way it was originally set up matched the structure of the `merchant` `show` page so I would have left that as it was.